tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/remove_invalid_procedure_source_records_test.py

```diff
@@ -0,0 +1,149 @@
+"""
+Integration test for remove_invalid_procedure_source_records module
+
+Original Issues: DC-1210
+"""
+
+#Python Imports
+import os
+from datetime import date
+
+# Third party imports
+from dateutil import parser
+
+# Project Imports
+from app_identity import PROJECT_ID
+from common import JINJA_ENV, PROCEDURE_OCCURRENCE, CONCEPT
+from cdr_cleaner.cleaning_rules.remove_invalid_procedure_source_records import RemoveInvalidProcedureSourceRecords
+from tests.integration_tests.data_steward.cdr_cleaner.cleaning_rules.bigquery_tests_base import BaseTest
+
+PROCEDURE_SOURCE_CONCEPT_IDS_TEMPLATE = JINJA_ENV.from_string("""
+INSERT INTO 
+    `{{project_id}}.{{dataset_id}}.{{procedure_occurrence_table}}`
+(procedure_occurrence_id, person_id, procedure_concept_id, procedure_date, procedure_datetime,
+ procedure_type_concept_id, modifier_concept_id, quantity, provider_id, visit_occurrence_id,
+ visit_detail_id, procedure_source_value, procedure_source_concept_id, modifier_source_value)
+
+VALUES
+    (51380225, 4, 4000000, '2009-04-27', TIMESTAMP('2009-04-27'), 38000270, 0, 1, 0, NULL,NULL,'92014', 4000000, NULL),
+    (76392641, 5, 5000000, '2011-05-19', TIMESTAMP('2011-05-19'), 38000270, 0, 1, 0, NULL,NULL,'99243', 5000000, NULL),
+    (22888767, 6, 5000000, '2007-12-23', TIMESTAMP('2007-12-28'), 38000270, 0, 1, 0, NULL,NULL,'99243', 5000000, NULL),
+
+    (20074825, 1, 1000000, '2012-09-23', TIMESTAMP('2012-09-23'), 44786631, 0, 1, 0, NULL,NULL,'99458', 2514636, NULL),
+    (12557074, 2, 2000000, '2010-06-21', TIMESTAMP('2010-06-21'), 47876139, 0, 1, 0, NULL,NULL,'45734', 4578984, NULL),
+    (56145455, 3, 3000000, '2011-02-14', TIMESTAMP('2011-02-14'), 45454515, 0, 1, 0, NULL,NULL,'48445', 8545787, NULL)
+
+""")
+
+CONCEPT_IDS_TEMPLATE = JINJA_ENV.from_string("""
```

Comment:
There is this function `copy_vocab_tables` available for tests. It loads the vocab tables for you, so you do not have to create vocab tables with dummy values each time.

https://github.com/all-of-us/curation/blob/e9105b626f140d5bfab21130716b0ca1b729c097/tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/bigquery_tests_base.py#L337